apps/beacon/main.cpp

```diff
@@ -180,19 +181,20 @@ int main()
       {
         member->beacon_service.StartNewCabinet(cabinet, static_cast<uint32_t>(cabinet.size() / 2),
                                                block_number, block_number + aeon_length,
-                                               static_cast<uint64_t>(std::time(nullptr)));
+                                               static_cast<uint64_t>(std::time(nullptr)),
```

Comment:
Not in the scope of this PR but we should add a wrapper around this I think and update all the parts of the code base (this part being the UTC clock getter)

Reply:
Todo noted